src/dataaccess/IDataSource.jsx

```diff
@@ -13,6 +13,23 @@ class IDataSource {
     savePatient(patient) {
         throw new Error("savePatient not implemented by " + this.constructor.name);
     }
+
+    // On the origins of this term see: https://en.wikipedia.org/wiki/Gestalt_(Mac_OS)
+    getGestalt() { 
+        // 
+        // Shape of the gestalt : { 
+        //      requestTypes: {
+        //          sync: bool
+        //          asynch: bool

```

Comment:
Quick note that the comment here says `asynch` but everywhere else in the code it's `async`.

Reply:
Taken care of! 👍